src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -676,6 +679,17 @@ protected void setBelowMaxDepthTransactionLimit(int maxAnalyzedTransactions) {
         this.maxAnalyzedTransactions = maxAnalyzedTransactions;
     }
 
+    @Override
+    public int getPoWThreads() {
+        return powThreads;
+    }
+
+    @JsonProperty
+    @Parameter(names = "--pow-threads", description = PearlDiverConfig.Descriptions.POW_THREADS)
+    protected void setPoWThreads(int powThreads) {
```

Comment:
change to `setPowThreads`
```suggestion
    protected void setPowThreads(int powThreads) {
```
Jackson knows how to translate the name to snake_case. This is why ini wasn't working

Reply:
Ah, thanks so much, that fixed it.